src/osd/osd_types.h

```diff
@@ -1137,6 +1138,7 @@ struct pg_pool_t {
     case FLAG_NOSCRUB: return "noscrub";
     case FLAG_NODEEP_SCRUB: return "nodeep-scrub";
     case FLAG_FULL_QUOTA: return "full_quota";
+    case FLAG_FULL_QUOTA_OBJECTS: return "full_quota_objects";
```

Comment:
Should we rename FLAG_FULL_QUOTA to FLAG_FULL_QUOTA_BYTES?  it will be slightly inconsistent if the flag is already set when we upgrade, but i think the mon will almost immediately readjust the flags to reflect the current stats, right?  We can make the compat encoding bitwise-or the two flags into one for older clients.  But new clients would need to be updated to check for both flags.  :/

Reply:
My thought was actually not to do that, because we don’t want to update all the different pieces that do fullness checks to examine both flags, or to deal with forward-and-backwards versioning so much. This lets us just check for the extra OBJECTS flag when doing the health outputs since in all other cases we want the same behavior anyway.

Reply:
yeah, makes sense!  a lot of effort for what amounts to flag cosmetics

Reply:
I guess then I would just adjust hte FLAG_FULL_QUOTA comment to say the pool object *or* byte quota

Reply:
Which comment are you looking at? I don't think FULL_QUOTA specifies anywhere since it's always been either one.